src/style.css

```diff
@@ -0,0 +1,127 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: 300;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: 800;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+.text__input::placeholder {
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  font-weight: 300;
+  line-height: 21.86px;
+  color: #3d4e61;
+}
+
+.form {
+  margin-top: 20px;
+  width: 100%;
+  height: 70px;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  position: relative;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+}
+
+.form--small {
+  width: 100%;
+  height: 42px;
+}
+
+.form:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.form:active {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
+
+.form__label {
+  width: 100%;
+  height: 100%;
+  position: absolute;
+}
+
+.text__input {
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  position: absolute;
+  background: transparent;
+  font-weight: 800;
+  color: #3d4e61;
+  border: 0;
+  left: 61px;
+  top: 25px;
+  bottom: 25px;
+}
```

Comment:
Your input must occupy all available container space, not only default.
![Screenshot 2023-01-26 at 13 52 39](https://user-images.githubusercontent.com/54940595/214830276-b83eb88e-dfb5-493f-8756-bd3df99bdd0a.png)